GitUI/Editor/RebaseTodoHighlightingStrategy.cs

```diff
@@ -164,7 +164,7 @@ private static bool TryHighlightInteractiveRebaseCommand(IDocument document, Lin
 
             return false;
 
-            bool IsHexChar() => char.IsDigit(c) || (c >= 'a' && c <= 'f');
+            bool IsHexChar() => char.IsDigit(c) || c is >= 'a' and <= 'f';
```

Comment:
Might need parens
```suggestion
            bool IsHexChar() => char.IsDigit(c) || c is (>= 'a' and <= 'f');
```

Reply:
it doesnt. but if that is your preference i will add them

Reply:
If not necessary, omit the parens. `(2 * 3) + 4` is overkill to me - but required.

Reply:
We should add parens here to make the intent obvious. [`&&` has the precedence over `||`](https://learn.microsoft.com/en-us/dotnet/csharp/language-reference/operators/#operator-precedence), however, a lot of developers aren't aware of this, and I personally fixed too many of such issues. IIRC, there's a Roslyn analyser for this case too.

Reply:
Oh, we've changed from `&&` to `and` - I should've woken up by now and be able to read properly :)